stage1/init/kvm/capabilities.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kvm
+
+import (
+	"strings"
+	"math"
+
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/appc/spec/schema/types"
+)
+
+// FindCapabilities reads all capabilitites from app isolators
+// returned int64 contain an information about required capabilities
+func FindCapabilities(isolators types.Isolators) (caps int64) {
+	for _, i := range isolators {
+		switch v := i.Value().(type) {
+		case types.LinuxCapabilitiesSet:
```

Comment:
if name i.Value() name ""os/linux/capabilities-retain-set"
or else "print not implemtened !" - as required by spec